server.R

```diff
@@ -91,541 +85,594 @@ server <- function(input, output, session) {
 
   output$uiScreenButton <- renderUI({
     if (values$dataUploaded && values$metaUploaded) {
-      actionButton("screenbutton", "Screen files", width = "80%")
+      if (values$environment == "shinyapps") {
+        actionButton("external_screenbutton", "Screen files", width = "80%")
+      } else {
+        actionButton("screenbutton", "Screen files", width = "80%")
+      }
     }
   })
 
-  observeEvent(input$screenbutton, {
-    shinyjs::hideElement(id = "guidance")
-
-    # Show loading screen ---------------------------------------------------------------------------------
-    shinyjs::showElement(id = "loading", anim = TRUE, animType = "fade", time = 1)
-
-    values$shouldShow <- TRUE
-
-    # Disable buttons ---------------------------------------------------------------------------------
-
-    disable("uiScreenButton")
-    disable(selector = "span[class='btn btn-default btn-file']")
+  # Confirmation modal for external use ----------------------------------------------------------
+
+  observeEvent(input$external_screenbutton, {
+    shinyalert::shinyalert(
+      title = "",
+      text = "This is the external version of the DfE data screening app. We do not control the servers that this app runs on. You should only use this to screen published or dummy data. For sensitive files, you should clone the repository and screen files locally instead.",
```

Comment:
Could you make this stand out more? Add bold and a few line breaks maybe?

Reply:
Have done this in the latest commit @lauraselby